src/Control/Carrier/Lift.hs

```diff
@@ -1,4 +1,6 @@
 {-# LANGUAGE FlexibleInstances, GeneralizedNewtypeDeriving, MultiParamTypeClasses #-}
+-- | Provides a carrier over a monad @m@ that allows monadic actions to be lifted
+-- into a larger context with 'Control.Effect.Lift.sendM'.
```

Comment:
1. We should mention that it’s a carrier for `Lift m`.

2. `sendM` should be in scope.